pulpcore/pulpcore/app/viewsets/custom_filters.py

```diff
@@ -10,3 +13,31 @@ class CharInFilter(filters.BaseInFilter, filters.CharFilter):
     than one object in a single query.
     """
     pass
+
+
+class HyperlinkRelatedFilter(filters.Filter):
+    """
+    Enables a user to filter by a foreign key using that FK's href
+    """
+
+    def filter(self, qs, value):
+        """
+        Args:
+            qs (django.db.models.query.QuerySet): The Queryset to filter
+            value (string): href representing an instance of foreign key
+
+        Returns:
+            Queryset filtered by the foreign key instance
+        """
+
+        if not value:
+            return qs
```

Comment:
this seems now obsoleted by line 39 but I'm wondering the same: shouldn't a 400/404 be raised in case empty value in here?